gradle/dependencies.gradle

```diff
@@ -8,27 +8,28 @@ ext {
   ]
 
   version = [
-      mapboxMapSdk       : '5.2.0-beta.4',
-      mapboxServices     : '2.2.9',
-      autoValue          : '1.4.1',
-      autoValueParcel    : '0.2.5',
-      junit              : '4.12',
-      supportLibVersion  : '26.1.0',
-      constraintLayout   : '1.0.2',
-      mockito            : '2.10.0',
-      hamcrest           : '2.0.0.0',
-      errorprone         : '2.0.21',
-      butterknife        : '8.8.1',
-      leakCanaryVersion  : '1.5.4',
-      timber             : '4.5.1',
-      awsPolly           : '2.3.8',
-      testRunnerVersion  : '1.0.1',
-      espressoVersion    : '3.0.1',
-      spoonRunner        : '1.6.2',
-      commonsIO          : '2.5',
-      robolectric        : '3.4.2',
-      lifecycleCompiler  : '1.0.0',
-      lifecycleRuntime   : '1.0.3'
+      mapboxMapSdk     : '5.2.0-beta.4',
```

Comment:
To avoid conflicts with Lost, we should bump `mapboxMapSdk` version to `5.2.0`.

> Per https://github.com/mapbox/mapbox-gl-native/pull/9488 the location provider is optional (we're not including Lost within the SDK anymore) and in https://github.com/mapbox/mapbox-gl-native/pull/10355 we bumped the Lost version to `3.0.4` https://github.com/mapbox/mapbox-gl-native/blob/ffea8e145f5b67bbeb7aec46c77d73b95ef6ea31/platform/android/dependencies.gradle#L23 which is the one that MAS `2.2.9` uses.